src/python/pants/backend/docker/subsystems/dockerfile_parser_test.py

```diff
@@ -204,6 +210,7 @@ def test_baseimage_tags(rule_runner: RuleRunner) -> None:
                 "FROM gcr.io/tekton-releases/github.com/tektoncd/operator/cmd/kubernetes/operator:"
                 "v0.54.0@sha256:d1f0463b35135852308ea815c2ae54c1734b876d90288ce35828aeeff9899f9d\n"
                 "FROM $PYTHON_VERSION AS python\n"
+                "FROM python:$VERSION\n"
```

Comment:
I find this behaviour odd, but wanted to make sure I was consistent here.

Reply:
It's also not obvious to me why we keep `build-arg` like we do instead of keeping them in separate collections? Since we could prepend `build-arg:` we could just as well have appended to a different list and save us the trouble?

Reply:
The value that is used in our interpolation becomes the literal string `$VERSION`